models/source/im-bm-ci/im_ontap_format.sql

```diff
@@ -0,0 +1,25 @@
+{{
+  config({
+    "materialized" : "table",
+    "post-hook" : [
+      "grant select on table {{this}} to group non_gl_read_only"
+      ]
+    })
+}}
+
+select
+    udf_ontap_format_id::int as ontap_format_id ,
+    udf_format_desc as format_desc,
+  case
+    when udf_liquid_ounces = '' then null
+    else udf_liquid_ounces::int
+  end as liquid_ounces,
+  case
+    when udf_sub_unit_format = '' then null
+    else udf_sub_unit_format::int
+  end as sub_unit_format,
```

Comment:
Use the nullif() function, and rename this field to sub_unit_format_id instead of sub_unit_format